### DISCUSSION TITLE
Doc for payloadExtraction is wrong

### DISCUSSION THREAD

**Discussion by nicodotcomputer** [nuxt/nuxt#19861] [General]
Hi y'all!

I was a bit surprised to see that now Nuxt turns on the payload extraction by default and that the setting is hidden in the experimental settings.

Anyway, there's a typo in the doc where it says that the default value is `false` but in reality it's `true`.

Cheers

https://nuxt.com/docs/api/configuration/nuxt-config#payloadextraction

**Comment by danielroe** [comment]
The default value is `false`, unless you are calling `nuxt generate` in which case you will see the following printed in your terminal:

```
Nuxt 3.3.1 with Nitro 2.3.1

 WARN  Using experimental payload extraction for full-static output. You can opt-out by setting experimental.payloadExtraction to false.
```
